src/content/docs/get-started/storefront-compatibility.mdx

```diff
@@ -0,0 +1,127 @@
+
+## Release information
+
+**Version:** 4.7.0-beta14
+**Release date:** September 3, 2024
+
+## Major features
+
+This package defines the following queries and mutations:
+
+* `confirmCancelOrder` mutation - Cancels the specified order for a guest.
+
+* [`estimateTotals` mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/estimate-totals/) - Estimates totals for cart based on the address.
+
+* [`generateCustomerToken` mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/generate-token/) - Creates a new customer token.
+
+* [`recaptchaFormConfig` query](https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/recaptcha-form-config/) - Returns information about the reCaptcha configuration for the specified form type.
+
+* `resendConfirmationEmail` mutation - Resends a confirmation email to the specified customer upon request.
+
+## Installation
+
+You must have Adobe Commerce 2.4.7 installed. Magento Open Source is not supported.
+
+<Tabs>
+  <TabItem label="Cloud infrastructure">
+
+  Use this method to install the Storefront Compatibility Package using the Cloud infrastructure.
+
+    1. On your local workstation, change to the project directory for your Adobe Commerce on cloud infrastructure project.
+      
+    1. Check out the environment branch to update using the Adobe Commerce Cloud CLI.
+    
+       ```bash
+       magento-cloud environment:checkout <environment-id>
+       ```
+        
+    1. Add the Storefront Compatibility module.
+
+       ```bash
+       composer require adobe-commerce/storefront-compatibility
+       ```
+       
+    1. Update package dependencies.
+    
+       ```bash
+       composer update "adobe-commerce/storefront-compatibility"
+       ```
+       
+    1. Commit and push code changes for the `composer.json` and `composer.lock` files.
+    
+    1. Add, commit, and push the code changes for the `composer.json` and `composer.lock` files to the cloud environment
+    
+       ```bash
+       git add -A
+       git commit -m "Add module"
+       git push origin <branch-name>
+       ```
+       
+       Pushing the updates to the cloud environment initiates the [Commerce cloud deployment process](https://experienceleague.adobe.com/en/docs/commerce-cloud-service/user-guide/develop/deploy/process) to apply the changes. Check the deployment status from the [deploy log](https://experienceleague.adobe.com/en/docs/commerce-cloud-service/user-guide/develop/test/log-locations#deploy-log).
+
+  </TabItem>
+  
+  <TabItem label="On premises">
+
+  Use this method to install the Storefront Compatibility Package for an on-premises instance.
+
+1. Use Composer to add the package module to your project:
+
+   ```bash
+   composer require adobe-commerce/storefront-compatibility
+   ```
+
+1. Update dependencies and install the extension:
+
+   ```bash
+   composer update  "adobe-commerce/storefront-compatibility"
```

Comment:
I don't think it makes sense to run `update  "adobe-commerce/storefront-compatibility"` after `require adobe-commerce/storefront-compatibility` as require is installing the latest available package version